### PR TITLE
API

### PR DESCRIPTION
**Pull request by Bhekisile**
Creating a game and storing its Id
Create a function to get all scores for a game
Create a function to save a score for a game

### CODE REVIEW

File: module/saveScore.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      body: JSON.stringify({ user: name, score }),
+    });
+    if (!response.ok) {
+      throw new Error('Failed to save score');
+    }
+    const { result } = await response.json();
+    return result;
+  } catch (error) {
+    // console.error(error);
```
Comment (afanimartin) [inline module/saveScore.js]: - Please remove any comments from your code. Comments make our code ugly and disorganized

File: module/getScores.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const getScores = async (gameId) => {
+  try {
+    const response = await fetch(`${apiUrl}games/${gameId}/scores/`);
+    if (!response.ok) {
+      throw new Error('Failed to fetch game scores');
+    }
+    const { result } = await response.json();
+    return result;
+  } catch (error) {
+    // console.error(error);
```
Comment (afanimartin) [inline module/getScores.js]: - Please remove this and any other comments you have

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Bhekisile, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Use of professional readme template
- App runs and functions as expected
- Use of arrow functions
- Use of `async` and `await` with the `fetch` method to send and receive data.

## Required Changes ♻️
_Check the comments under the review._
- The MIT license link leads to a 404 page. Please fix that
- Live demo link leads to `www.google.com`. You can simply use `N/A` if the app is not deployed. Please fix that also

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Learn to properly format the readme file [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Bhekisile 👋👋👋, 

tryagain


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅

- [x] Linters are passing👍.
- [x] Commented block of code have been taken away👍.


 ## Required Changes ♻️

- [ ] Just as requested by the previous reviewer, your `mit` license link is still broken🥲. please kindly ensure to fix it as this gives your project a less professional look👍.
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Bhekisile , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] [OPTIONAL] there is no content in your Future Features section of your README kindly fix that

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**